RealmApp.tsx

```diff
@@ -1,5 +1,6 @@
 import Realm from "realm";
 import { APP_ID } from "@env";
-const app = new Realm.App({ id: APP_ID });
+
+const app = new Realm.App({ id: "cultural-survival-app-ugsqf" });
```

Comment:
Maybe we should try to set up .env if we can? We have a bit of a susceptibility here.

Reply:
That's a great thought, I'll test it right now to see if env works on mine.